AvgHR.py

```diff
@@ -27,6 +27,8 @@ def avghr_unbound(times, voltages):
 
 
 def avghr(times, voltages, t1=0, t2=float('inf')):
+    if times == [] or voltages == []:
```

Comment:
nit: you may want to check the `len` of times and voltages to see if they're zero instead of comparing to empty array, as that's typically more canonical and less error prone